po/de.po

```diff
@@ -0,0 +1,271 @@
+
+#. TRANSLATORS: Don't translate this text
+#: data/dev.tchx84.Portfolio.metainfo.xml.in:30
+msgid "Martin Abente Lahaye"
+msgstr ""
+
+#: src/window.ui:49
+msgid "Filter"
+msgstr "Filter"
+
+#: src/window.ui:72
+msgid "Show Hidden Files"
+msgstr "Verborgene Dateien anzeigen"
+
+#: src/window.ui:121
+msgid "Sort"
+msgstr "Sortieren"
+
+#: src/window.ui:144
+msgid "A-Z"
+msgstr "A-Z"
+
+#: src/window.ui:170
+msgid "Last Modified"
+msgstr "Zuletzt Geändert"
+
+#: src/window.ui:226
+msgid "Help"
+msgstr "Hilfe"
+
+#: src/window.ui:242
+msgid "About Portfolio"
+msgstr "Über Portfolio"
+
+#: src/window.ui:713
+msgid "No files found"
+msgstr "Keine Dateien gefunden"
+
+#: src/window.ui:1170
+msgid "Properties"
+msgstr "Eigenschaften"
+
+#: src/window.ui:1231
+msgid "Name"
+msgstr "Name"
+
+#: src/window.ui:1278
+msgid "Location"
+msgstr "Ort"
+
+#: src/window.ui:1325
+msgid "Type"
+msgstr "Typ"
+
+#: src/window.ui:1372
+msgid "Size"
+msgstr "Größe"
+
+#: src/window.ui:1419
+msgid "Created"
+msgstr "Erstellt"
+
+#: src/window.ui:1466
+msgid "Modified"
+msgstr "Geändert"
+
+#: src/window.ui:1513
+msgid "Accessed"
+msgstr "Zugegriffen"
+
+#: src/window.ui:1591
+msgid "About"
+msgstr "Über"
+
+#: src/window.ui:1751
+msgid "<a href=\"https://github.com/tchx84/portfolio\">Website</a>"
+msgstr "<a href=\"https://github.com/tchx84/portfolio\">Webseite</a>"
+
+#: src/window.ui:1806
+msgid ""
+"This program comes with absolutely no warranty. See the <a href=\"https://"
+"www.gnu.org/licenses/gpl-3.0.html\">GNU General Public License, version 3 or "
+"later</a> for details."
+msgstr ""
+"Dieses Programm kommt OHNE JEDWEDE GARANTIE.\n"
+"Besuchen Sie <a href=\"https://www.gnu.org/licenses/gpl-3.0.html\">GNU "
+"General Public License, version 3 or later</a> für weitere Informationen."
+
+#: src/window.ui:1835
+msgid ""
+"Portfolio logo is a modifed version of Jorgen Carlberg's work. The original "
+"version can be found <a href=\"https://commons.wikimedia.org/wiki/File:"
+"Portfolio.svg\"> here </a>, licensed under <a href=\"https://creativecommons."
+"org/licenses/by/3.0/deed.en\"> CC BY 3.0 </a>."
+msgstr ""
+"Das Portfoliologo ist eine modifizierte Variante von Jorgen Carlbergs "
+"Arbeit. Die Orginalversion kann <a href=\"https://commons.wikimedia.org/wiki/"
+"File:Portfolio.svg\"> hier </a> gefunden werden. Lizenziert unter <a href="
+"\"https://creativecommons.org/licenses/by/3.0/deed.en\"> CC BY 3.0 </a>."
+
+#: src/window.ui:1873
+msgid "Created by"
+msgstr "Erstellt von"
+
+#: src/window.ui:1904
+msgid "Artwork by"
+msgstr "Künstlerische Darstellung von"
+
+#: src/window.ui:1935
+msgid "Translated by"
+msgstr "Übersetzung von"
+
+#: src/window.ui:1970
+msgid "Contributions by"
+msgstr "Erstellt von"
+
+#: src/places.py:78
+msgid "Places"
+msgstr "Orte"
+
+#: src/places.py:82
+msgid "Devices"
+msgstr "Geräte"
+
+#: src/places.py:92
+msgid "Home"
+msgstr "Persönlicher Ordner"
+
+#: src/places.py:137
+msgid "System"
+msgstr ""
```

Comment:
It is System in german as well ?

Reply:
Oh... my fault. Sorry. Changed it to Rechner.